src/core/load-fragment.c

```diff
@@ -4212,6 +4212,16 @@ int config_parse_emergency_action(
 
         r = parse_emergency_action(rvalue, MANAGER_IS_SYSTEM(m), x);
         if (r < 0) {
+                if (r == -EOPNOTSUPP && MANAGER_IS_USER(m)) {
+                        /* Compat mode: remove for systemd 241. */
+
+                        log_syntax(unit, LOG_INFO, filename, line, 0,
```

Comment:
Ah, I copied the code from `DEFINE_CONFIG_PARSE_ENUM`. It is correct there, because the parsing functions return -1, but here I have a valid errno. Will fix.